README.md

```diff
@@ -1,2 +1,78 @@
 # kube-sqs-autoscaler
-Kubernetes pod autoscaler based on queue size in AWS SQS
+Kubernetes pod autoscaler based on queue size in AWS SQS. It periodically retrieves the number of messages in your queue and scales pods accordingly.
+
+## Setting up
+Setting up kube-sqs-autoscaler requires two steps:
+1) Deploying it as an incluster service in your cluster
+2) Adding AWS permissions so it can read the number of messages in your queues.
```

Comment:
Agreed, in the future when we have custom metrics setup it should roughly look like that